meson.build

```diff
@@ -7,9 +7,21 @@ i18n = import('i18n')
 # Set our translation domain
 add_global_arguments('-DGETTEXT_PACKAGE="@0@"'.format (meson.project_name()), language:'c')
 
+config_data = configuration_data()
+# Provide a path of the directory where translation binary files (*.mo) are located
+config_data.set_quoted('LOCALEDIR', get_option('prefix') / get_option('localedir'))
+# Provide translation domain name
+config_data.set_quoted('GETTEXT_PACKAGE', meson.project_name())
```

Comment:
If this is the same as project ID can we just assume this is also always the same as application ID? It feels like kinda overkill to use a config file for this if we know it's equal to a variable we're already using in the application class